src/ejabberd_c2s.erl

```diff
@@ -3350,3 +3380,9 @@ setup_accum(Acc, StateData) ->
     Server = StateData#state.server,
     mongoose_acc:update(Acc, #{server => Server, user => User}).
 
+%% @doc This function is executed when c2s receives a stanza from TCP connection.
+-spec from_sender_element(jlib:xmlel(), StateData :: state()) -> mongoose_acc:t().
+from_sender_element(El, #state{sid = SID, jid = JID}) ->
+    Acc = mongoose_acc:from_element(El),
+    Acc1 = mongoose_acc:add_prop(sender_sid, SID, Acc),
+    mongoose_acc:add_prop(sender_jid, JID, Acc1).
```

Comment:
In my PR #1797 I put JID to `user_jid` key in the acc. Would be good to use the same name, I'm not sure which one is better suited.

Reply:
Offtop: It's what my problem with accs. Nobody knows how to use them. 
Ontop: sender_jid is a property, so it would be never stripped. And nobody would change it.
user_jid is an accumulator field, so it would be stripped, the moment we resend accumulator to another process.

These accumulator fields are not stripped `from, from_jid, to, to_jid, persistent_properties, global_distrib`. Properties are not stripped.

I can't use `from_jid`, because it's the same as `From :: #jid{}`. I.e. it's different for request and reply.

`sender_jid` semantics is that it would be the same for the whole live of accumulator. I.e. we set it once, and never touch it again.

We touch `from_jid`, when we reply.

Reply:
Thanks for the clarifications. Looks like I'll adjust my code in already merged #1797 to use `sender_jid` after this PR is merged. There is no need to have duplicate keys if they hold the same value.